fdbserver/include/fdbserver/StorageMetrics.actor.h

```diff
@@ -255,7 +255,9 @@ Future<Void> serveStorageMetricsRequests(ServiceType* self, StorageServerInterfa
 				self->metrics.getReadHotRanges(req);
 			}
 			when(SplitRangeRequest req = waitNext(ssi.getRangeSplitPoints.getFuture())) {
-				if (!self->isReadable(req.keys)) {
+				if ((!req.tenantInfo.hasTenant() && !self->isReadable(req.keys)) ||
```

Comment:
Found this existing bug in SplitRangeRequest for tenants, that the shard check did not include the tenant prefix if it was a tenant-enabled range stream @sfc-gh-etschannen